nle/env/base.py

```diff
@@ -460,7 +465,12 @@ def reset(self, wizkit_items=None):
         """
         self._episode += 1
         new_ttyrec = self._ttyrec_pattern % self._episode if self.savedir else None
-        self.last_observation = self.env.reset(new_ttyrec, wizkit_items=wizkit_items)
+        if self._episode % self._episode_save_cycle == 0:
```

Comment:
We can/should probably roll that into the logic of the line before.

Reply:
yeah i just commented on my own repo - we can do that too